packages/core/src/utils.js

```diff
@@ -343,6 +347,23 @@ export async function withRetries(fn, { count, onRetry, signal, throwOn }) {
   }
 }
 
+export function redactSecrets(message) {
```

Comment:
we are calling this function multiple times, it doesnt make sense to read patterns from file everytime

Reply:
Also why is this function sync ? this will block execution of other parts of cli

Reply:
Now executing this method at very last so should be fine as till that time processing would be over.